public/js/components/Editor.js

```diff
@@ -160,20 +164,29 @@ const Editor = React.createClass({
     }
   },
 
+  renderWelcomeBox() {
+    const modifierTxt = Services.appinfo.OS === "Darwin" ? "Cmd" : "Ctrl";
+    return dom.div(
+      { className: "welcomebox" },
+      `Want to find a file? (${modifierTxt} + P)`
+    );
+  },
+
   render() {
     const breakpoints = this.props.breakpoints.valueSeq();
 
     return (
       dom.div(
-        { className: "editor-wrapper devtools-monospace" },
-        dom.div({ className: "editor-mount" }),
+        { className: "editor-wrapper" },
+        dom.div({ className: "editor-mount devtools-monospace" }),
         breakpoints.map(bp => {
           return Breakpoint({
             key: makeLocationId(bp.location),
             breakpoint: bp,
             editor: this.editor && this.editor.codeMirror
           });
-        })
+        }),
+        !this.props.selectedSource ? this.renderWelcomeBox() : null
```

Comment:
Thanks, I'd be happy to add an empty center header later for consistency.

My concern is that this introduces bloat to the Editor. I'd like to keep the keep the editor focused on the source.